docs/application/dotnet/get-started/overview.md

```diff
@@ -1,33 +1,35 @@
 # Get Started with .NET Applications
 
-When you create your first Tizen .NET application, you can select from the following options that uses C# as the programming language:
+When you create your first Tizen .NET application, you can select from the following options that use C# as the programming language:
 
 -   [Wearable .NET application](wearable/first-app.md)
 
-    A wearable application is targeted to run on a smart watch. The application can be optimized for the smaller screen.
+    Applications targeted to run on smartwatches. It can be optimized for small screens.
```

Comment:
In this page, we have a list which lists up URLs to each "Create your first app" and descriptions for the URLs.
With this PR, the singular subjects in the descriptions are changed to plural.

However I'm not sure using plural here, while there is only one example in each "Create your first app" pages.

For example in line 5, 
\[Wearable .NET application](wearable/first-app.md) page is creating "an application", and the description sentence for this page is calling it as "Applications".

Another confusing point is that, these "Applications" are called as "It", not "They", in the following sentence.
> Applications ... It can be optimized for small screens.

How about checking the plural/singular used in the sentences again? In my thought singular form is required for each descriptions. Would you please let me know your thought?

Reply:
Thank you for pointing this.
I will use singular form.

Reply:
@mijin85cho 
This can be changed to:
An application targeted to run on smartwatches. It can be optimized for small screens.
or 
We can combine and rephrase as:
An application targeted to run on smartwatches and can be optimized for small screens.

Please verify and let me know your opinion on this.

Reply:
@mijin85cho 
Thank you for the suggestion.
Actually, the usage of "target" in both cases is fine. It is the difference in writing style. I can take your suggestion. However, "An applications" is the wrong usage. Therefore, to make it clear, I will rephrase as:

An application that is targeted to run on smartwatches. It can be optimized for small screens.
Please verify.

Reply:
@Reni-90 Thanks for fixing my typo in the suggestion. Also I think your suggestion is acceptable to apply.

I hope you apply the update to this style for other description lines, 
for the sentences in overview for dotnet, web and native also,
to get the consistency between the pages.

Reply:
@mijin85cho 
Thank you very much for your feedback. 
I will use the same style for all other pages to maintain consistency.